### DISCUSSION TITLE
symbol._internal._plus  ?

### DISCUSSION THREAD

**Discussion by haibo-qiu** [apache/incubator-mxnet#13608] [Q&A]
I have read the lstmcell code in mxnet. However, I have a question.

The following code is the part of lstmcell:

next_c = symbol._internal._plus(forget_gate * states[1], in_gate * in_transform,
name=’%sstate’%name)
next_h = symbol._internal._mul(out_gate, symbol.Activation(next_c, act_type=“tanh”),
name=’%sout’%name)

I want to know the detail of “symbol._internal._plus” and the difference between it and “mxnet.symbol.broadcast_plus”, but I don’t find it in the mxnet document.
Could any one explain it for me? Thanks so much!!

**Comment by vdantu** [comment]
@mxnet-label-bot add [question]

**Comment by vdantu** [comment]
@haibo-qiu : Could you post this to discuss.mxnet.io ? I think you will catch wider audience's attention there.
